src/contracts/core/RewardsCoordinator.sol

```diff
@@ -274,6 +278,47 @@ contract RewardsCoordinator is
         emit ClaimerForSet(earner, prevClaimer, claimer);
     }
 
+    /**
+     * @notice Sets the commission for a specific operator for a specific operatorSet (avs, operatorSetId)
+     * @param operator The operator address
+     * @param avs The address of the AVS
+     * @param operatorSetId The operatorSetId of the AVS
+     * @param commissionBips The commission in bips for the operator, must be <= MAX_COMMISSION_BIPS
+     * @return effectTimestamp The timestamp at which the operator commission update will take effect
+     * @dev Only callable by the operator
+     */
+    function setOperatorCommissionBips(
+        address operator,
```

Comment:
Why do we need to provide operator address? We assume `msg.sender == operator`?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/627/commits/68478ee59925e8e2cbb5fb1bfc348079fe6cb614